dataframe-jupyter/src/main/kotlin/org/jetbrains/kotlinx/dataframe/jupyter/Integration.kt

```diff
@@ -169,11 +168,6 @@ internal class Integration(private val notebook: Notebook, private val options:
             if (enableExperimentalCsv?.toBoolean() == true) {
                 println("CSV module is already enabled by default now.")
             }
-            if (enableExperimentalGeo?.toBoolean() == true) {
-                println("Enabling experimental Geo module: dataframe-geo")
```

Comment:
instead of completely removing it, please provide a helpful message of what to do instead :)

We mentioned the `enableExperimentalGeo` in some release, so if it suddenly does nothing anymore, people will be confused

Reply:
Fixed